README.md

```diff
@@ -25,13 +42,22 @@ cd ..
 flask run  # Run in root directory
 ```
 
-After logging in, use https://127.0.0.1:5000/admin to modify entries in the database.
+After logging in, use [localhost admin](https://127.0.0.1:5000/admin) to modify entries in the database.
 
 
-## Full setup
-```bash
-Note: you should have docker + docker-compose installed on your computer
+### Full setup
+This setup will create the following items:
+* application - lms code
+* middleware (messaging queue) - rabbit-mq
+* persistence database - postgresql
+
+Steps to do:<br />
+1. clone this repo
```

Comment:
*T*itle first word, add `.` at the end of each sentence, use full words (repository, application).